INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+
+We provide a `setup.cfg
+<https://raw.githubusercontent.com/matplotlib/matplotlib/master/setup.cfg.template>`_
+file that goes with :file:`setup.py` which you can use to customize
+the build process. For example, which default backend to use, whether
+some of the optional libraries that Matplotlib ships with are
+installed, and so on.  This file will be particularly useful to those
+packaging Matplotlib.
+
+If you have installed prerequisites to nonstandard places and need to
+inform Matplotlib where they are, edit ``setupext.py`` and add the base
+dirs to the ``basedir`` dictionary entry for your ``sys.platform``.
+e.g., if the header to some required library is in
+``/some/path/include/someheader.h``, put ``/some/path`` in the
+``basedir`` list for your platform.
+
+.. _install_requirements:
+
+Dependencies
+------------
+
+Matplotlib requires a large number of dependencies:
+
+  * `Python <https://www.python.org/downloads/>`_ (>= 2.7 or >= 3.4)
+  * `numpy <http://www.numpy.org>`_ (>= |minimum_numpy_version|)
+  * `setuptools <https://setuptools.readthedocs.io/en/latest/>`__
+  * `dateutil` (>= 1.1)
+  * `pyparsing <https://pyparsing.wikispaces.com/>`__
+  * `libpng <http://www.libpng.org>`__ ( >= 1.2) 
+  * `pytz <http://pytz.sourceforge.net/>`__
+  * FreeType (>= 2.3)
+  * `cycler <http://matplotlib.org/cycler/>`__ (>= 0.10.0)
+  * `six <https://pypi.python.org/pypi/six>`_
+  * `backports.functools_lru_cache <https://pypi.python.org/pypi/backports.functools_lru_cache>`_
+    (for Python 2.7 only)
+  * `subprocess32 <https://pypi.python.org/pypi/subprocess32/>`_ (for Python
+    2.7 only, on Linux and mac os X only)
+
+Optionally, you can also install a number of packages to enables better user
+interface toolkit. See :ref:`what-is-a-backend` for more details on the
+optional Matplotlib backends and the capabilities they provide.
+
+  * :term:`tk` (>= 8.3, != 8.6.0 or 8.6.1): for the TkAgg backend;
+  * `PyQt4 <https://pypi.python.org/pypi/PyQt4>`_ (>= 4.4): for the Qt4Agg backend;
+  * `PyQt5 <https://pypi.python.org/pypi/PyQt5>`_: for the Qt5Agg backend;
+  * :term:`pygtk` (>= 2.4): for the GTK and the GTKAgg backend;
+  * :term:`wxpython` (>= 2.8 or later): for the WX or WXAgg backend;
+  * `pycairo <https://pypi.python.org/pypi/pycairo>`_;
+  * `PySide <https://pypi.python.org/pypi/PySide>`_;
+  * `PyGTK <https://pypi.python.org/pypi/PyGTK>`_;
+  * `Tornado <https://pypi.python.org/pypi/tornado>`_: for the WebAgg backend.
+
+For better support of animation output format and image file format, LaTeX,
+etc you can
+install the following:
+
+  * `ffmpeg <https://www.ffmpeg.org/>`__/`avconv
+    <https://libav.org/avconv.html>`__ or `mencoder
+    <http://www.mplayerhq.hu/design7/news.html>`__ (for saving movies);
+  * `ImageMagick <http://www.imagemagick.org/script/index.php>`__ (for saving
+    animated gifs);
+  * `Pillow <http://python-pillow.org/>`__ (for a larger selection of image
+    file format: JPEG, BMP, and TIFF image files);
```

Comment:
That would be surprising. I think it is for IO, but both reading and writing though I have no clue on that one.